src/renderers/shaders/ShaderChunk/morphtarget_vertex.glsl.js

```diff
@@ -1,17 +1,18 @@
 export default /* glsl */`
 #ifdef USE_MORPHTARGETS
 
-	transformed += ( morphTarget0 - position ) * morphTargetInfluences[ 0 ];
-	transformed += ( morphTarget1 - position ) * morphTargetInfluences[ 1 ];
-	transformed += ( morphTarget2 - position ) * morphTargetInfluences[ 2 ];
-	transformed += ( morphTarget3 - position ) * morphTargetInfluences[ 3 ];
+	transformed *= morphTargetBaseInfluence;
```

Comment:
Any chances to add a link to the formula of this code? Or maybe to a reference that explains what's going on in the next lines? I guess that would make the code more clear and accessible for the interested reader.

Reply:
I've updated both shaders (position / normal) and JS-side code to explain this a bit better.